ui/imports/shared/views/chat/MessageContextMenuView.qml

```diff
@@ -208,7 +208,7 @@ StatusMenu {
 
     StatusAction {
         id: downloadImageAction
-        text: qsTr("Download image")
+        text: (root.imageSource.endsWith(".gif")) ? qsTr("Copy GIF") : qsTr("Copy image")
```

Comment:
This should be "Download ...", not copy/pasted from above ;)

Reply:
lol you caught me! done!